cmd/lncli/cmd_open_channel.go

```diff
@@ -560,7 +574,9 @@ func openChannelPsbt(rpcCtx context.Context, ctx *cli.Context,
 			// remove it again, this would just produce another
 			// error.
 			cancelErr := chanfunding.ErrRemoteCanceled.Error()
-			if err != nil && strings.Contains(err.Error(), cancelErr) {
+			if err != nil && strings.Contains(err.Error(),
```

Comment:
I think we'd prefer,

```suggestion
			if err != nil && strings.Contains(
			    err.Error(), cancelErr, 
			    ) {
```